apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -5,15 +5,23 @@ import { loadSqlEquiv, queryRows, queryRow, queryAsync } from '@prairielearn/pos
 
 import * as b64Util from '../../lib/base64-util.js';
 import { getCourseFilesClient } from '../../lib/course-files-api.js';
-import { QuestionGenerationContextEmbeddingSchema, QuestionSchema } from '../../lib/db-types.js';
+import {
+  type Issue,
+  QuestionGenerationContextEmbeddingSchema,
+  QuestionSchema,
+} from '../../lib/db-types.js';
+import { getAndRenderVariant } from '../../lib/question-render.js';
 import { type ServerJob, createServerJob } from '../../lib/server-jobs.js';
+import { selectCourseById } from '../../models/course.js';
+import { selectQuestionById } from '../../models/question.js';
+import { selectUserById } from '../../models/user.js';
 
 import { createEmbedding, openAiUserFromAuthn, vectorToString } from './contextEmbeddings.js';
 import { validateHTML } from './validateHTML.js';
 
 const sql = loadSqlEquiv(import.meta.url);
 
-const MODEL_NAME: OpenAI.Chat.ChatModel = 'gpt-3.5-turbo';
+const MODEL_NAME: OpenAI.Chat.ChatModel = 'gpt-4o';
```

Comment:
As @mwest1066 suggested in another recent PR, let's pin this to a specific version (probably the latest as of this moment).

Reply:
changed in merge.

Reply:
@jlevine18 this wasn't resolved. I'm going to let this merge as-is so this can be deployed, but please double-check in the future before marking things as resolved.